src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,63 @@
 package core.basesyntax;
 
+import static java.lang.Integer.parseInt;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int DATE_POSITION = 0;
+    private static final int NAME_POSITION = 1;
+    private static final int HOUR_POSITION = 2;
+    private static final int SALARY_PER_HOUR_POSITION = 3;
+    private static final String SEPARATOR = " - ";
+    private static final String REGEX = " ";
+    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder stringBuilder = new StringBuilder("Report for period ");
+        stringBuilder.append(dateFrom)
```

Comment:
```suggestion
        StringBuilder stringBuilder = new StringBuilder("Report for period ")
                                  .append(dateFrom)
```
you can append right away, if you wish